react/what-is-props/what-is-props.md

```diff
@@ -0,0 +1,131 @@
+# What is props?
+
+---
+
+If you are beginner in react you must have heard about `props` a lot, so let's understand, how is it so important to understand.
+
+## How is it useful?
+
+Before starting we need to understand how react works. So React is a component based `JavaScript` Library where we can divide our project into various reusable components.
+
+For example the below code is your main `App.js` file which you can see when we create a project using create-react-app
+
+```JavaScript
+import "./styles.css";
+import Greet from "./Greet";
+
+function App() {
+  var Humans = [
+    {
+      id: 1,
+      name: "Ashray",
+      age: 22
+    },
+    {
+      id: 2,
+      name: "Aman",
+      age: 25
+    }
+  ];
+
+  return (
+    <div className="App">
+      <Greet Humans={Humans} />
+    </div>
+  );
+}
+export default App;
+```
+
+In the above code we can see that I have created one `Component`.
+
+- Greet
+
+There is an array of object's with key value pair of `id`, `name` and `age`
```

Comment:
**"There is an array of object's with key-value pair of `id`, `name` and `age`"**, this can be rephrased as

**"There is an array `humans` with property `id`, `name`, `age`"**

We are trying to make the text absolutely clear. You can assume that if someone is coding in react, they will know what a key-value pair is. So we can refer them as properties which is much easier to read.